server/controllers/auth.controller.js

```diff
@@ -57,12 +62,53 @@ function updatePassword(req, res, next) {
     const user = req.user;
     user.password = req.body.password;
     user.save()
-        .then(() => {
-            return res.sendStatus(httpStatus.OK)
-        })
+        .then(() => res.sendStatus(httpStatus.OK))
         .catch(error => next(error));
         // TODO should we produce a new token? A user could change the password
         // and change it again without re-authenticating
 }
 
-export default { login, logout, updatePassword };
+/**
+ * Returns 200 OK if password was reset successfully
+ * @param req
+ * @param res
+ * @param next
+ * @returns {*}
+ */
+function resetToken(req, res, next) {
+    const userLine = 'You have requested the reset of the password for your GFTrade account';
```

Comment:
Why does this say GFTrade account?

Reply:
Good catch, it was just a copypasta from me.